tests/10-EdvClient.spec.js

```diff
@@ -16,6 +16,106 @@ describe('EdvClient', () => {
     await mock.server.shutdown();
   });
 
+  it('should only contain one indexed document after 3 updates', async () => {
+    const getLatestDocsByIndexHas = async (client, invocationSigner, index) => {
+      const indexedDocs = await client.find({
+        has: index,
+        invocationSigner
+      });
+      const uniqueIds = [...new Set(indexedDocs.map(item => item.id))];
+
+      const docsFromGet = await Promise.all(
+        uniqueIds.map(id => {
+          return client.get({id, invocationSigner});
+        })
+      );
+      return docsFromGet;
+    };
+
+    const client = await mock.createEdv();
+    client.ensureIndex({attribute: 'content.indexedKey'});
+    const testId = await EdvClient.generateId();
+    const doc = {
+      id: testId,
+      content: {someKey: '111111', indexedKey: 'value1'}
+    };
+    let version1 = await client.insert({
+      doc,
+      invocationSigner,
+      keyResolver
+    });
+    version1 = await client.get({id: doc.id, invocationSigner});
+
+    // console.log("version: 1", version1);
```

Comment:
nite: you may wish to strip my lazy console logs

Reply:
Nit should be resolved: https://github.com/digitalbazaar/edv-client/pull/18/commits/91e6ac4cf2d27970276c9e42eec616268dd7c13f